src/components/MobileMenuBar.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'styled-components'
+import logo from '../assets/logo.svg'
+import menu from '../assets/menu.svg'
+import { mediaQueries } from './Common';
+
+const MobileMenuBarContainer = styled.div`
+  display: none; 
+  padding: 20px;
```

Comment:
Any reason this isn't inside the media query?

Reply:
No reason, I'll add it to the media query! + Will decrease the padding 👍